src/components/Item/Item.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, ReactNode } from 'react';
+import { ResponseItem } from '../../types';
+import classes from './Item.module.scss';
+import platformsSlugData from '../../utils/platformsSlugData';
+
+export default class Item extends Component<{ item: ResponseItem }> {
+  changeClassName = (slug: string) => {
+    const platformsSlug = platformsSlugData;
```

Comment:
Why do you need this assignment? Is it possible just to use platformsSlugData?
`const currentClassName = platformsSlugData[slug] || 'platform_template';`

Reply:
you're totally right. fixed